Multisig/UI/Settings/OwnerKeyManagement/ImportOwnerKey/KeyPickerController/KeyPickerController.swift

```diff
@@ -79,7 +79,7 @@ class KeyPickerController: UITableViewController {
         vc.trackingEvent = .enterKeyName
         vc.placeholder = "Enter name"
         vc.address = privateKey.address
-        vc.badgeName = KeyType.deviceImported.imageName
+        vc.badgeName = KeyType.deviceGenerated.imageName
```

Comment:
I would use .walletConnect type instead

Reply:
This screen opens for keys, that were generated on device from a seed phrase. Why use .walletConnect type here?

Reply:
why this change? this should have been deviceImported, this key is imported from a seed phrase or a private key. It's not a wallet connect type.